### PR TITLE
added VSTest.Next project

### PR DESCRIPTION
**Pull request by heejaechang**
Currently vstest.next project is empty. 

I just made sure it builds without any dependency issues caused by mixing dev14 and dev15 dependencies.

another change includes moving VS.Next to under Core directory so that it sits with test.next project.

### CODE REVIEW

**Comment by heejaechang** [comment]
@panopticoncentral can you take a look?

File: src/VisualStudio/Core/Next/Diagnostics/OutOfProcDiagnosticAnalyzerExecutor.cs [hunk 1]
```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```
Comment (heejaechang) [inline src/VisualStudio/Core/Next/Diagnostics/OutOfProcDiagnosticAnalyzerExecutor.cs]: hmmm this is also move. not sure why it shows up as new file. might be due to I pulled upstream master which has change on this file?

**Comment by panopticoncentral** [comment]
:+1:

**Comment by heejaechang** [comment]
added internalvisiblestotest to new vstest.next project

**Pull request merged**